requirements/system/work-machine-client.tex

```diff
@@ -1,22 +1,31 @@
-\section{Work machine client(WMC)}
-\subsection{Main features}
-\begin{enumerate}
-  \item Each work machine shall have a single instance of WMC running.
-  \item On startup, WMC shall connect to the server and report its hardware capabilities. (TODO: define hw capabilities in glossary?)
-  \item WMC must be able to receive commands from the central server.
-  \item WMC shall support the following commands:
+\section{Worker client(WC)}
+  \subsection{Functional requirements}
+    \subsubsection{Main features}
+    \begin{enumerate}
+      \item[WCF10] On startup, WC shall connect to the server and report its hardware capabilities.
+      \item[WCF20] WC must be able to receive commands from the central server.
+      \item[WCF30] Each work machine must have a single instance of WC running.
+      \item[WCF40] WC shall support the following commands:
+      \begin{enumerate}
+        \item[WCF41] Start/Resume execution of a job.
+        \item[WCF42] Pause execution of a job.
+        \item[WCF43] Remove a job from the list of running jobs on the current machine.
+        \item[WCF44] Query the current state of the work machine: running jobs, load on hardware.
+        \item[WCF45] Terminate this instance of the WC.
+      \end{enumerate}
+      \item[WCF50] WC shall download the job data to the local storage if needed. (TODO: need to decide when)
+      \item[WCF60] WC shall notify the server whenever a job is completed.
+    \end{enumerate}
+
+    \subsubsection{Optional features}
+      \begin{enumerate}
+        \item[WCFO10] WC shall notify the server periodically on the progress of the job.
+        \item[WCFO20] WC shall decide what kind of local storage to use for each job. (RAM, SSD or HDD).
+        \item[WCFO30] WC shall estimate the time of arrival of a job.
+      \end{enumerate}
+
+  \subsection{Product Data}
   \begin{enumerate}
-    \item Start/Resume execution of a job.
-    \item Pause execution of a job.
-    \item Remove a job from the list of running jobs on the current machine.
-    \item Query the current state of the work machine: running jobs, load on hardware.
-    \item Terminate this instance of WMC.
-  \end{enumerate}
-  \item WMC shall download the job data to the local storage if needed. (TODO: need to decide when)
-  \item WMC shall notify the server whenever a job is completed.
-\end{enumerate}
-\subsection{Optional features}
-  \begin{enumerate}
-    \item WMC shall notify the server periodically on the progress of the job.
-    \item WMC shall decide what kind of local storage to use for each job. (RAM, SSD or HDD).
+    \item[WCPD10] Docker containers for all the jobs running on this machine.
```

Comment:
Just `WCP` (we decided on those here: https://github.com/DistributedTaskScheduling/documentation/blob/master/general/Structure.md)